index.js

```diff
@@ -140,9 +142,15 @@ module.exports = ({utPort}) => class SchedulePort extends utPort {
 
     addJob(name, job) {
         if (!this.jobs[name]) {
+            if (this.jobs[name]) {
+                this.stopCron(name);
+            }
             this.log.info && this.log.info({opcode: 'Schedule', msg: `Add Job ${name}`, job: job});
             this.jobs[name] = new cron.CronJob({
                 cronTime: job.pattern,
+                unrefTimeout: function() {
```

Comment:
Are you sure this has to be a function? It seems to me this is just a boolean. 
Also I think this option does not exist in the version of cron module we are using. I will whitelist the latest version.
If we use this option, there is no need to stop the cron jobs explicitly.

Reply:
I saw here more implementation on this. But did not very our version
https://www.npmjs.com/package/cron